DevEdu.API/DevEdu.DAL/Repositories/CourseRepository.cs

```diff
@@ -45,11 +46,36 @@ public void DeleteCourse(int id)
 
         public CourseDto GetCourse(int id)
         {
-            return _connection.QuerySingleOrDefault<CourseDto>(
+            return _connection.Query<CourseDto, TopicDto, MaterialDto, TaskDto, GroupDto, CourseDto>(
```

Comment:
do we really need all these entities at the same time?)

Reply:
It'll be next task

Reply:
I've kept only group in related, couse topic, material and task have more common sense